includes/reader-activation/sync/class-metadata.php

```diff
@@ -0,0 +1,397 @@
+<?php
+/**
+ * Reader Activation Sync Metadata.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Reader_Activation\Sync;
+
+use Newspack\Reader_Activation\Sync;
+use Newspack\Reader_Activation\ESP_Sync;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Metadata Class.
+ */
+class Metadata extends Sync {
```

Comment:
I don't see why this class needs to extend Sync. Right now the only thing it's using from the parent class is the `log` method... but that's not enough and could be a class on its own... a helper method... or even a trait

Reply:
You're right! I believe at some point during the refactor some more inherited methods were being used but as I worked abstracting it further it became more self-contained.

We talked about it being a trait, but it doesn't really have to, being a static class. We would also lose the possibility of having the consts in `Metadata`.

Removed in d22dd80cf82f03ebe88b42d7f811cc7a0eacaefc

Reply:
I meant the Logger being a trait, so you can call `self::log`, and it could add a prefix to the log based on a static property, so each class would have its own prefix

But no need for that, we have a Logger class already